kms/src/test/java/com/example/SnippetsIT.java

```diff
@@ -17,269 +17,480 @@
-    }
+  public void testCreateKeyAsymmetricSign() throws IOException {
+    new CreateKeyAsymmetricSign()
+        .createKeyAsymmetricSign(PROJECT_ID, LOCATION_ID, KEY_RING_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created asymmetric key");
+  }
+
+  @Test
+  public void testCreateKeyHsm() throws IOException {
+    new CreateKeyHsm().createKeyHsm(PROJECT_ID, LOCATION_ID, KEY_RING_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created hsm key");
+  }
 
-    // at least one version should be enabled, so we should never make it here
-    assertTrue("no versions are enabled", false);
+  @Test
+  public void testCreateKeyLabels() throws IOException {
+    new CreateKeyLabels().createKeyLabels(PROJECT_ID, LOCATION_ID, KEY_RING_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created key with labels");
   }
 
   @Test
-  public void disableCryptoKeyVersion_disables() throws Exception {
-    CryptoKeyVersion version =
-        Snippets.createCryptoKeyVersion(PROJECT_ID, LOCATION_ID, KEY_RING_ID, CRYPTO_KEY_ID);
+  public void testCreateKeyRing() throws IOException {
+    new CreateKeyRing().createKeyRing(PROJECT_ID, LOCATION_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created key ring");
+  }
 
-    String versionId = parseVersionId(version.getName());
+  @Test
+  public void testCreateKeyRotationSchedule() throws IOException {
+    new CreateKeyRotationSchedule()
+        .createKeyRotationSchedule(PROJECT_ID, LOCATION_ID, KEY_RING_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created key with rotation");
+  }
 
-    CryptoKeyVersion disabled =
-        Snippets.disableCryptoKeyVersion(
-            PROJECT_ID, LOCATION_ID, KEY_RING_ID, CRYPTO_KEY_ID, versionId);
-    assertThat(disabled.getState()).isEqualTo(CryptoKeyVersion.CryptoKeyVersionState.DISABLED);
+  @Test
+  public void testCreateKeySymmetricEncryptDecrypt() throws IOException {
+    new CreateKeySymmetricEncryptDecrypt()
+        .createKeySymmetricEncryptDecrypt(PROJECT_ID, LOCATION_ID, KEY_RING_ID, getRandomId());
+    assertThat(stdOut.toString()).contains("Created symmetric key");
   }
 
   @Test
-  public void enableCryptoKeyVersion_enables() throws Exception {
-    CryptoKeyVersion version =
-        Snippets.createCryptoKeyVersion(PROJECT_ID, LOCATION_ID, KEY_RING_ID, CRYPTO_KEY_ID);
+  public void testCreateKeyVersion() throws IOException {
+    new CreateKeyVersion().createKeyVersion(PROJECT_ID, LOCATION_ID, KEY_RING_ID, SYMMETRIC_KEY_ID);
+    assertThat(stdOut.toString()).contains("Created key version");
+  }
 
-    String versionId = parseVersionId(version.getName());
+  // @Test
+  // public void testDecryptAsymmetric() throws IOException {
+  //   new DecryptAsymmetric()
+  //       .decryptAsymmetric(
+  //           PROJECT_ID, LOCATION_ID, KEY_RING_ID, ASYMMETRIC_DECRYPT_KEY_ID, "1", ciphertext);
+  //   assertThat(stdOut.toString()).contains("Created key version");
+  // }
```

Comment:
Can we uncomment this? Lets either use `Ignore` annotation with a reason if it need to not run, otherwise leave it in.

Reply:
Done